crates/biome_css_parser/Cargo.toml

```diff
@@ -19,6 +19,7 @@ biome_js_unicode_table = { workspace = true }
 biome_parser           = { workspace = true }
 biome_rowan            = { workspace = true }
 tracing                = { workspace = true }
+unicode-bom = "2.0.3"
```

Comment:
Since we are about to use this crate in multiple crates, we usually add a dependency at the root `Cargo.toml`, and then here you'll have

```toml 
unicode-bom = { workspace = true }
```

Reply:
i didn't realize this was a feature! that's pretty neat. i'll update it.